notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2842 @@
+{
```

Comment:
<p>Let's add a note that this exercise is involved. To discuss: is "estimated time to complete" a useful metric, and if so, how long should we suggest?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
<p>Work out cutting out some of the repeated things, still warn about level of complexity/time investment</p><p> </p>